application/src/main/kotlin/net/corda/v5/application/uniqueness/model/UniquenessCheckError.kt

```diff
@@ -0,0 +1,51 @@
+package net.corda.v5.application.uniqueness.model
```

Comment:
Having seen the corresponding [corda-runtime-os PR](https://github.com/corda/corda-runtime-os/pull/1952/files), I actually wonder if this is in the right package - do we really want our backing store and uniqueness checker components and libraries to be dependent on `corda-application`? Conceptually that feels a bit back to front.

Maybe it should live in its own `uniqueness` module, similar to other services such as ledger, membership, persistence etc.

Reply:
Yeah that's a valid point!